validation/policies/io/konveyor/forklift/ovirt/tpm_test.rego

```diff
@@ -0,0 +1,17 @@
+package io.konveyor.forklift.ovirt
+ 
+test_without_tpm_enabled {
+    mock_vm := { "name": "test",
+                 "tpmEnabled": false
+                }
+    results = concerns with input as mock_vm
+    count(results) == 0
+}
+
+test_with_tpm_enabled {
+    mock_vm := { "name": "test",
+                 "tpmEnabled": true
```

Comment:
aren't we supposed to check it with certain operating system types?

Reply:
yup, my bad. i only changed the rule and not the test.

Reply:
@liranr23 and how come that the test passed then?..

Reply:
> @liranr23 and how come that the test passed then?..

what I meant to ask is how come that we didn't see a failure in the CI?

Reply:
our makefile executing `go test ./pkg/... ./cmd/... -coverprofile cover.out`
however for the validation we need something like `opa test`. we don't test our validation test units in the CI. we need to fix that.